frontend/src/app/components/version-selector/version-selector.component.spec.ts

```diff
@@ -22,4 +31,6 @@ describe('VersionSelectorComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  //TODO stub unit tests
```

Comment:
Looking at file content, it's probably obvious it was generated automatically and is missing any actual content, or at least stubs. We can probably drop this comment.